e2e/tests/company/administrator/settings/workspace.spec.ts

```
@@ -0,0 +1,71 @@
+    });
+
+    expect(updatedCompany?.publicName).toBe("Updated Company Name");
+    expect(updatedCompany?.website).toBe("https://example.com");
+    expect(updatedCompany?.brandColor).toBe(testColor.toLowerCase());
+  });
+
+  test("displays initial company data in form fields", async ({ page }) => {
+    const companyNameInput = page.getByLabel("Company name");
+    const websiteInput = page.getByLabel("Company website");
+
+    await expect(companyNameInput).toHaveValue(company.name as string);
```

ESLint

[error] Do not use any type assertions.

(@typescript-eslint/consistent-type-assertions)